### PR TITLE
Align credentials stores part 13

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Relates:[ link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5274)

## Description
Change the type of credentials, from type 4 to type 9.

## Minimum version of Cortex XSOAR
- [x] 6.5.0

## Does it break backward compatibility?
  - [x] No

### CODE REVIEW

File: Packs/Pwned/Integrations/PwnedV2/PwnedV2.py [hunk 1]
```diff
@@ -64,7 +67,7 @@ def http_request(method, url_suffix, params=None, data=None):
             wait_amount = wait_regex.group()
         else:
             demisto.error('failed extracting wait time will use default (5). Res body: {}'.format(res.text))
-            wait_amount = 5
+            wait_amount = '5'
```
Comment (Shellyber) [inline Packs/Pwned/Integrations/PwnedV2/PwnedV2.py:70]: Why this change?

File: Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py [hunk 2]
```diff
@@ -1115,6 +1115,8 @@ def export_vulnerabilities_command(args: Dict[str, Any]) -> PollResult:
 
 
 def main():  # pragma: no cover
+    if not (ACCESS_KEY and SECRET_KEY):
+        return_error('Access key and secret key must be provided.')
```
Comment (Shellyber) [inline Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py:1119]:
```suggestion
        return_error('Access Key and Secret Key must be provided.')
```

File: Packs/Pwned/Integrations/PwnedV2/PwnedV2.py (hunk 1 above)
Comment (maimorag) [reply Packs/Pwned/Integrations/PwnedV2/PwnedV2.py:70]: lint errors

**Comment by maimorag** [comment]
pwnd test fail- existing problem [link to jira issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-1630)

File: Packs/Pwned/Integrations/PwnedV2/PwnedV2.py (hunk 1 above)
Comment (Shellyber) [reply Packs/Pwned/Integrations/PwnedV2/PwnedV2.py:70]: Please revert this. And check why the lint fails. Maybe the declaration was wrong from the beginning.
There is a difference between string and int - timeout seems to be an int to me

**Comment by Shellyber** [comment]
> pwnd test fail- existing problem [link to jira issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-1630)

It seems that we have credentials for this.
Did you check why the test failed? tried to fix it?

**Comment by maimorag** [comment]
fails since the Tenable.io account had expired - [link to Jira ticket](https://jira-hq.paloaltonetworks.local/browse/CRTX-86110)

File: Packs/Pwned/Integrations/PwnedV2/PwnedV2.py (hunk 1 above)
Comment (Shellyber) [reply Packs/Pwned/Integrations/PwnedV2/PwnedV2.py:69]: What about this? are you sure about this change?

File: Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py [hunk 3]
```diff
@@ -139,8 +139,8 @@
     'Critical']
 
 BASE_URL = demisto.params()['url']
-ACCESS_KEY = demisto.params()['access-key']
-SECRET_KEY = demisto.params()['secret-key']
+ACCESS_KEY = demisto.params().get('credentials_key', {}).get('identifier') or demisto.params()['access-key']
```
Comment (Shellyber) [inline Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py:142]: Access key is not secret?

**Review by Shellyber — APPROVED** [review]
